content/en/synthetics/guide/email-validation.md

```diff
@@ -13,59 +13,56 @@ further_reading:
 
 ## Overview
 
-Classic web application journeys often involve emails being triggered and sent to a users’ mailbox, such as email verification after account creation, email sent to reset forgotten passwords, order confirmations, or mail confirmation after contact form submission.
+Web application journeys often involve emails being triggered and sent to users' mailboxes, such as an email verification after account creation, an email sent to reset forgotten passwords, an email sent to notify order confirmation, or an email confirmation after contact form submission.
 
-Making sure your application’s email mechanisms are working as expected is key to ensuring a great user experience on your website.
-
-Datadog [Browser Tests][1] allow you to:
-
-- Confirm an email was correctly sent following up on an in-app trigger
-- Check an email's content
-- Click on the links located within the sent emails in order to navigate to other URLs and validate your whole flows, including web and email steps
-
-To perform email validation inside a Datadog Browser test:
+Maintaining a great user experience on your website includes ensuring that your application's email mechanisms are working properly. 
 
 ## Create an email variable
 
-After filling your Browser test details (start URL, devices, locations, frequency, notification), navigate to **Save Details & Record Test**, select **Variables** and then select **Email** from the email variable dropdown list:
+To add an email variable called `EMAIL`:
+
+1. Click **Variables** and select **Email** from the dropdown menu. 
+2. Click **Add Variable** to make the variable available for you to use when you start recording. 
 
 {{< img src="synthetics/guide/email-validation/adding-variable.mp4" alt="Create an email variable" video="true"  width="100%">}}
 
-In the above example, an email variable called `EMAIL` is created. The newly created email variable generates a unique mailbox maintained by Datadog at every test execution. This is key to avoid any conflicts among your Browser test runs.
+The email variable generates a unique mailbox maintained by Datadog at every test execution, which enables your browser tests to run without conflicts.
 
 ## Record steps
 
-In the upper left corner of the UI, click the **Start Recording** button and record the steps leading to the email being triggered using your freshly created email variable. Use the hand icon to input variables in the text inputs of the form.
+Once you have created an email variable, you can [confirm the email was sent correctly](#confirm-the-email-was-sent) after an in-app trigger. First, check the email's content, click on links located in sent emails to [navigate to other URLs](#navigate-through-links-in-an-email), and then validate the entire user flows, including web and email steps. 
```

Comment:
I am not sure about this sentence. Let's discuss about it during our sync

Reply:
```suggestion 
```